Makefile

```diff
@@ -88,10 +88,6 @@ k8s: validate ## Build default K8s version of EKS Optimized AL2 AMI
 
 # Build dates and versions taken from https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 
-.PHONY: 1.21
```

Comment:
Can we also remove 1.21 from line# 40?

Reply:
good catch, I believe we should remove it